### PR TITLE
docs: update global-rule.md

### PR DESCRIPTION
**Pull request by lustlost**
添加global rule优先级的说明

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Hi, please update https://github.com/lustlost/apisix/blob/patch-1/docs/en/latest/architecture-design/global-rule.md too :) 

BTW, we need add description for this PR :)

File: docs/zh/latest/architecture-design/global-rule.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ curl -X PUT \
 如上所注册的 `limit-count` 插件将会作用于所有的请求。
 
 我们可以通过以下接口查看所有的 `GlobalRule`:
+当 `Route` 和 `Service` 都开启同一个插件时，Route 参数的优先级是高于 Service 的。但是`GlobalRule`优先级高于`Route`和`Service`
```
Comment (spacewander) [inline docs/zh/latest/architecture-design/global-rule.md:48]: I think this sentence can be moved behind the example.

Comment (tzssangglass) [reply docs/zh/latest/architecture-design/global-rule.md:48]: I think the `Route 参数的优先级是高于 Service 的` here is not accurate, it should be `APISIX 只会执行 Route 上的插件，Service 上的插件被覆盖`, and the `但是 GlobalRule 优先级高于 Route 和 Service` can change to `GlobalRule 上的插件一定会执行，无论 Route 和 Service 上是否开启同一个插件`, what do you think @spacewander

**Review by tokers — APPROVED** [review]

File: docs/zh/latest/architecture-design/global-rule.md (hunk 1 above)
Comment (moonming) [reply docs/zh/latest/architecture-design/global-rule.md:48]: Is there a consensus here?

Comment (tzssangglass) [reply docs/zh/latest/architecture-design/global-rule.md:48]: hi, @lustlost, any news?

Comment (liaomengge) [reply docs/zh/latest/architecture-design/global-rule.md:48]: > I think the `Route 参数的优先级是高于 Service 的` here is not accurate, it should be `APISIX 只会执行 Route 上的插件，Service 上的插件被覆盖`, and the `但是 GlobalRule 优先级高于 Route 和 Service` can change to `GlobalRule 上的插件一定会执行，无论 Route 和 Service 上是否开启同一个插件`, what do you think @spacewander
赞同

**Comment by tzssangglass** [comment]
ping @lustlos, do you have time to update this PR?

**Comment by moonming** [comment]
Can we merge this PR first？

**Comment by spacewander** [comment]
Let's wait for the author to update it.

**Pull request closed without merging**